src/aci/aci_build_F.cc

```diff
@@ -1182,7 +1182,7 @@ AdaptiveCI::get_excited_determinants_batch_vecsort(SharedMatrix evecs, SharedVec
                 double& V = pair.second;
                 if (V != 0.0) {
                     auto& det = pair.first;
-                    double delta = fci_ints_->energy_bit(det) - E0;
```

Comment:
Thanks for cleaning up this. I think it is a good solution to remove `energy_bit` and to have only one function.

Reply:
Ok, I just saw the code below and now I see that the new `energy(det)` is the efficient bit version for 64 bits and the standard one for larger orbital spaces.